src/VfsClient.js

```diff
@@ -33,15 +33,17 @@ function _readRawArchive(fs, darUrl) {
     let type = entry.attr('type')
     let content = fs.readFileSync(`${darUrl}/${entry.path}`)
     rawArchive[path] = {
+      encoding: 'utf8',
       type: type,
       data: content
     }
   })
   assets.forEach(asset => {
     let path = asset.attr('path')
     rawArchive[path] = {
+      encoding: 'url',
       type: 'image/jpg',
-      url: path
+      data: './data/kitchen-sink/'+path
```

Comment:
are you sure?

Reply:
Well, this is just for the dev setup, where we only have the kitchen sink example. ;) But yeah, if we wanted more, we need to change this.